index.html

```diff
@@ -0,0 +1,156 @@
+<!doctype html>
+<head>
+  <title>Static Web HTML and CSS Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+
+<body>
+
+
+<!-- 1st CARD --> 
+
+  <div class="card">
+
+    <header><h1 id=cardTitle>Yellow Kittens</h1></header>
+
+    <section>
+      <!-- The first section should contain three child block elements. -->
+      <img src="http://placekitten.com/g/400/400" class="product_image" alt="Yellow Kittens">
```

Comment:
So pretty much everyone tripped up on this. But technically the requirement asks that 'The first section should contain three child block elements.' and 5-i asks 'The first element contains the product image.'
It will be great practice to put your images in a block level element (such as a div or figure) and then try to recenter them.